src/workflows/workflows.ts

```diff
@@ -228,6 +229,14 @@ export class Workflows {
     );
   }
 
+  public prepareWithdrawalWithSigner(params: PrepareWithdrawalRequest) {
```

Comment:
can we have a request: {token, quantity} as a request, and {l1signer, l2Signer} as wallet.

```suggestion
  public prepareWithdrawalWithSigner(wallet: PrepareWithdrawalRequest, request: PrepareWithdrawal) 
```